or-model/src/main/java/org/openrefine/RefineModel.java

```diff
@@ -42,8 +42,9 @@ static public void registerClassMapping(String from, String to) {
     }
     
     static {
-        registerClassMapping("com.metaweb.*", "com.google.*");
-        registerClassMapping("com.google.gridworks.*", "com.google.refine.*");
+        registerClassMapping("com.metaweb.refine.*", "org.openrefine.*");
+        registerClassMapping("com.google.gridworks.*", "org.openrefine.*");
+        registerClassMapping("com.google.refine.*", "org.openrefine.*");
```

Comment:
I haven't looked into the issue closely given that I gave up on backwards compatibility with previous workspaces (see https://github.com/wetneb/OpenRefine/pull/335#issuecomment-1756921565). It's very well possible that somehow this class mapping mechanism has been broken by something else in the past and we just did not notice because it's not actually relied on unless someone is upgrading from a 10-year old version.
I could look into fixing it but I am not sure it's worth the effort, I'd rather make sure we don't use class names in workspace serialization in the new architecture (which should not be hard to reach).